### ISSUE TITLE
Files in Repository

### ISSUE THREAD

**Issue by RedRoboHood** [skylight-software/jonathan-windows#1]
Right now, there are a lot of files (binary or otherwise) that probably shouldn't be in the repository: project files (*.sln, *.suo), build files (everything in the bld and bin subdirectories), and maybe more. The problem with those files is that they are not:

- meant to be edited by humans
- necessary to build and run the program

Having such files in the repository results in bloat (they and all their revisions exist in the repository history forever) and conflicts (because many of them are generated files, different developers will accidentally commit their own versions of them), so they should instead be ignored by Git.

**Comment by miller9904** [comment]
The .sln file is how Visual Studio keeps track of project files.  If you don't have these files, you have to import the code as a new project every time.

**Comment by RedRoboHood** [comment]
Right, I was getting a bit mixed up there. [This SO question](http://stackoverflow.com/questions/1644653/which-files-in-a-visual-c-sharp-studio-project-dont-need-to-be-versioned) and several others address which files and folders should or should not be included in version control. As you said, the *.sln file(s) keep track of project information, so those should probably be included, but there are a _lot_ of other files that shouldn't be included (which I mentioned in the original post).
